.NET/Microsoft.Recognizers.Text.Number/Italian/Parsers/ItalianNumberParserConfiguration.cs

```diff
@@ -86,6 +86,19 @@ public override IEnumerable<string> NormalizeTokenSet(IEnumerable<string> tokens
                 }
             }
 
+            /*The following piece of code is needed to compute the fraction pattern number+'e mezzo'
```

Comment:
Nitpick: Please add the start and end multi-line comment markers on their own lines.